cylc/flow/cfgspec/globalcfg.py

```diff
@@ -826,16 +826,54 @@ def default_for(
                    range.
             ''')
             Conf('condemned', VDR.V_ABSOLUTE_HOST_LIST, desc=f'''
-                These hosts will not be used to run jobs.
+                List run hosts that workflows should *not* run on.
 
-                If workflows are already running on
-                condemned hosts, Cylc will shut them down and
-                restart them on different hosts.
+                These will be subtracted from the
+                `available <global.cylc[scheduler][run hosts]>`:
+
+                * Workflows will not start on condemned hosts.
+                * Workflows that are running on condemned hosts will attempt
+                  to migrate to an available host (providing the
+                  `auto restart
+                  <global.cylc[scheduler][main loop][auto restart]>`
+                  plugin is enabled).
+
+                This feature can be used to drain a host for patching, or
+                remove a host that is surplus to requirements.
+
+                If a hostname listed here is followed by a ``!`` character
+                ("force mode"), workflows running on it
+                will shutdown rather than attempting to
+                migrate (providing the
```

Comment:
```suggestion
                If a hostname listed here is followed by a ``!`` character
                workflows running on it will be told to shut down
                rather than migrate to another host (providing the
```